dashboards/template_dashboard.py

```diff
@@ -19,37 +22,29 @@ def template_dashboard(df, example_dropdown="option 1"):
     """Create and return the dashboard layout for display in the application."""
 
     barchart = bar_chart(df, "Category", "Value", color="Category")
-    barchart_dash = dcc.Graph(id="example bar chart", responsive=True, figure=barchart)
+    barchart_dash = dcc.Graph(
+        id="example bar chart", responsive="auto", figure=barchart
+    )
```

Comment:
What's the reason for switching True to "auto"? Not obviously related to the navbar, and doesn't match what we have in the housing supply dashboard, so feel it could use some explanation?

Reply:
That is the cause of the weird [styling bug](https://trello.com/c/WCEShj9D/188-%F0%9F%AA%B2-bug-%F0%9F%AA%B2-xl-visualisation-background-intermittently-disappears-on-selection-of-local-authority) we have where the graph loses its colour.

Reply:
Ah! Very nice. I guess we will see this in our dashboards soon then.